mmv1/templates/terraform/sweeper_file.go.tmpl

```diff
@@ -102,7 +141,7 @@ func testSweep{{ $.ResourceName }}(_ string) error {
 		resourceList, ok := res["{{ $.ResourceListKey }}"]
 		if !ok {
 			log.Printf("[INFO][SWEEPER_LOG] Nothing found in response.")
-			continue
+			return nil
```

Comment:
It looks like this got reverted by accident?

```suggestion
			continue
```

Reply:
ah thanks for this!